db/migrate/20151104115400_create_chargeback_tiers.rb

```diff
@@ -0,0 +1,13 @@
+class CreateChargebackTiers < ActiveRecord::Migration
+  def change
+    create_table :chargeback_tiers do |t|
+      t.bigint :chargeback_rate_detail_id
+      t.float :start
+      t.float :end
```

Comment:
We'll need to rename this column since it's easily confused with the `end` keyword.